### ISSUE TITLE
Upload: Keyboard focus and navigation

### ISSUE THREAD

**Issue by dlowenberg** [CDL-Dryad/dryad-product-roadmap#589]
Keyboard focus is not always visible. E.g., Unable to tell that focus is on Choose Files button.

Cannot navigate between radio buttons with only keyboard.

Can’t tab to the navigation buttons at the bottom of the page.
